starter/source/elements/sh3n/coquedk/cdkinit3.F

```diff
@@ -148,8 +148,8 @@ SUBROUTINE CDKINIT3(ELBUF_STR,GROUP_PARAM,
 C=======================================================================
       GBUF => ELBUF_STR%GBUF
       IORTHLOC = 0
-      IMAT  = IXTG(1,1+NFT)          ! mat N�
-      IPROP = IXTG(NIXTG-1,1+NFT)    ! property N�
+      IMAT  = IXTG(1,1+NFT)          ! mat N�
```

Comment:
can you correct the strange characters please ?

Reply:
Hi Sebastien, 

The strange characters in comment have been removed. 

Thanks a lot.